### DISCUSSION TITLE
Connecting to Multiple Database Servers

### DISCUSSION THREAD

**Discussion by surafi** [vendurehq/vendure#2042] [Q&A]
Does vendure support connecting to multiple database instances? Vendure timesout when connection to my database server and I wanted to do something like [this](https://orkhan.gitbook.io/typeorm/docs/multiple-data-sources)

**Comment by michaelbromley** [comment]
Hi,
In theory Vendure should support any TypeORM feature. Note however that the current 1.x versions are on an older version of TypeORM - 0.2.x. We are currently in the process of migrating over the TypeORM 0.3.x, which will be part of the upcoming Vendure v2.0 release.

But I'm still not sure about why you want this feature? You say the connection times out? How will multiple databases solve this?

**Reply by surafi** [reply]
I was thinking if multiple database servers could help because they will be sharing the load

**Reply by michaelbromley** [reply]
I'd say the first thing to try is vertical scaling - i.e. more RAM and CPU resources for your database server. It is much, much simpler if you can get by with a single database, and I'd guess that 95%+ sites can. What kind of hardware is your DB running on currently?

**Reply by surafi** [reply]
2 cores, 4GB RAM and 80GB SSD
